editor/editor_inspector.cpp

```diff
@@ -291,7 +302,7 @@ void EditorProperty::_notification(int p_what) {
 				} else {
 					draw_texture(checkbox, check_rect.position, color2);
 				}
-				int check_ofs = get_theme_constant(SNAME("hseparator"), SNAME("Tree")) + checkbox->get_width() + get_theme_constant(SNAME("h_separation"), SNAME("CheckBox"));
+				int check_ofs = checkbox->get_width() + get_theme_constant(SNAME("h_separation"), SNAME("Tree"));
```

Comment:
Is the removal of the CheckBox separation intentional?

I suppose this checkbox is used for stuff like Theme Overrides in control nodes:

![image](https://github.com/godotengine/godot/assets/4701338/3d3636f8-a941-45f9-8a39-b8af39baffc3)

Reply:
Just seems excessive in the context of EditorProperty because it's the only button in EditorProperty that you can independently control the "padding" of (i.e why only checkbox and not the revert button, key button, etc?), and also the way it was implemented is it was adding two constants together which I think makes theming less predictable